packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/ImageEdit.ts

```diff
@@ -296,6 +303,12 @@ export default class ImageEdit implements EditorPlugin {
 
             // Get initial edit info
             this.editInfo = getEditInfoFromImage(image);
+            if (this.image.height && this.image.width && this.image.style.maxWidth === '100%') {
```

Comment:
if height/width exists, wasResized should be true
Also need to check style.width and style.height.